src/components/BellNotificationDropDown/BellNotificationDropDown.js

```diff
@@ -44,6 +50,9 @@ const BellNotificationDropDown = () => {
 
   return (
     <NotificationCardWrapper>
+      <DismissButtonSecondary onClick={(event) => dismissAllNotifications(event)}>
```

Comment:
you can do `onClick={dismissAllNotifications}` too to avoid an anon function as a prop which I believe causes unnecessary rerenders. Its a nitpick though.

Reply:
will add it!